Submitted By:            Igor Živković <contact@igor-zivkovic.from.hr>
Date:                    2013-08-18
Initial Package Version: 1_54_0
Upstream Status:         Fixed
Origin:                  Upstream
Description:             Fixes compiling with Glibc-2.18.

diff -Naur boost_1_54_0.orig/boost/cstdint.hpp boost_1_54_0/boost/cstdint.hpp
--- boost_1_54_0.orig/boost/cstdint.hpp	2013-03-09 23:55:05.000000000 +0100
+++ boost_1_54_0/boost/cstdint.hpp	2013-08-18 21:45:19.223003705 +0200
@@ -41,7 +41,10 @@
 // so we disable use of stdint.h when GLIBC does not define __GLIBC_HAVE_LONG_LONG.
 // See https://svn.boost.org/trac/boost/ticket/3548 and http://sources.redhat.com/bugzilla/show_bug.cgi?id=10990
 //
-#if defined(BOOST_HAS_STDINT_H) && (!defined(__GLIBC__) || defined(__GLIBC_HAVE_LONG_LONG))
+#if defined(BOOST_HAS_STDINT_H)                                 \
+  && (!defined(__GLIBC__)                                       \
+      || defined(__GLIBC_HAVE_LONG_LONG)                        \
+      || (defined(__GLIBC__) && ((__GLIBC__ > 2) || ((__GLIBC__ == 2) && (__GLIBC_MINOR__ >= 17)))))
 
 // The following #include is an implementation artifact; not part of interface.
 # ifdef __hpux
